src/lang/po/de/Prusa-Firmware-Buddy_de.po

```diff
@@ -841,7 +841,7 @@ msgstr "Zum Kalibrieren mit  \nderzeit geladenem  \nFilament,          \ndrücke
 #. 9
 #: src/guiapi/src/window_msgbox.cpp:57
 msgid "TRY AGAIN"
-msgstr "NEU VERSUCHEN"
+msgstr "ERNEUT VERSUCHEN"
```

Comment:
This is a button label - Erneut versunchen would take the whole width of the screen to render ... can you think of something much shorter?

Reply:
NEUER VERSUCH
could work, which would mean
new attempt